### PR TITLE
Add end-user docs for user management

### PR DESCRIPTION
**Pull request by Kidswiss**

### CODE REVIEW

File: docs/modules/ROOT/pages/vshn-managed/mariadb/usermanagement.adoc [hunk 1]
```diff
@@ -0,0 +1,76 @@
+= Manage Users and Databases
+
+== Creating User and Databases
```
Comment (wejdross) [inline docs/modules/ROOT/pages/vshn-managed/mariadb/usermanagement.adoc:3]:
```suggestion
== Creating Users and Databases
```

File: docs/modules/ROOT/pages/vshn-managed/mariadb/usermanagement.adoc [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+<4> Write the connection secret to another namespace. If you want to connect from another namespace, please make sure that you configure the xref:vshn-managed/mariadb/security.adoc[allowed namespaces] accordingly.
+
+Please see the official https://mariadb.com/kb/en/grant/[MariaDB docs] for all available privileges. Only Grants applicable to databases are supported.
+
+== Deleting User and Databases
+To remove a user and its database, you need to remove it from the `access` array in the instance.
+
+A user or database will be removed from the instance, if there are no more references in the `access` array.
+
+.Example Deleting users and databases
```
Comment (wejdross) [inline docs/modules/ROOT/pages/vshn-managed/mariadb/usermanagement.adoc:49]:
```suggestion
.Example Deleting Users and Databases
```
just to keep it in the same manner as above

**Review by wejdross — APPROVED** [review]
small things

**Review by TheBigLee — APPROVED** [review]

**Comment by Kidswiss** [comment]
> We might also add a note that if an item is being updated from the access array that it will delete and recreate the user/database.

What do you mean by updated? The only time things get deleted is, if they get removed from the list completely.

**Comment by zugao** [comment]
> > We might also add a note that if an item is being updated from the access array that it will delete and recreate the user/database.
> 
> What do you mean by updated? The only time things get deleted is, if they get removed from the list completely.

What if the user edits `user1` into `user2` what will happen?

**Comment by Kidswiss** [comment]
> > > We might also add a note that if an item is being updated from the access array that it will delete and recreate the user/database.
> > 
> > 
> > What do you mean by updated? The only time things get deleted is, if they get removed from the list completely.
> 
> What if the user edits `user1` into `user2` what will happen?

Ah I see, I'll add something to explain that. Will also add it to PostgreSQL.

**Pull request merged**